CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+
+### Intentional Omission of Identity Validation
+
+Note that this CIP does not mention any kind of identity validation or verification for the members of the Constitutional Committee or the DReps.
+
+This is intentional.
+
+We hope that the community will strongly consider only voting for and delegating to those who provide something like a DID to make themselves known.
+However, enforcing identity verification is very difficult without some centralized oracle, which we consider to be a step in the wrong direction.
+
+### Initial Implementation Stage
+
+This document describes an ambitious change to Cardano governance. A two-stage implementation process will
+allow it to be rolled out more quickly, with some of the more complicated aspects
+(DReps and constitutional changes) rolled out in a second stage. This will also allow time
+for fuller evaluation of/consultation on incentives and other significant  issues, while establishing the basis for
+governance in Voltaire.
+Below is a high-level summary of an implementation plan for (very roughly) half of the plan.
+
+1. Add a subset of (important) governance actions to the transaction body (while deprecating protocol parameter updates and MIR certificates).
+  The following governance actions will be supported initially:
+  * protocol parameters
+  * hard forks
+  * withdrawals from the treasury
+  * Constitutional Committee changes
+2. Add votes to the transaction body, but disallow DRep votes.
+3. The first version of the new `RATIFY` rule will only count Constitutional Committee votes, plus SPO votes for hard forks (similarly to the previously defined, and now obsoleted, `CIP-47`).
+
+The remainder of this proposal will be implemented in the second stage.
+
+### Piggybacking on stake pool stake distribution
+
+The Cardano protocol is based on a Proof-of-Stake consensus mechanism, so using a stake-based governance approach is sensible. However, there are many ways by which one could define how to record the stake distribution between participants. As a reminder, network addresses can currently contain two sets of credentials: one to identify who can unlock funds at an address (a.k.a. payment credentials) and one that can be delegated to a stake pool (a.k.a. delegation credentials).
+
+Rather than defining a third set of credentials, we instead propose to re-use the existing delegation credentials, using a new on-chain certificate to determine the governance stake distribution. This implies that the DReps can (and likely will) differ from SPOs, so creating balance. On the flip side, it means that the governance stake distribution suffers from the same shortcomings as that for block production: for example, wallet software providers must support multi-delegation schemes and must facilitate the partitioning of stake into sub-accounts should an Ada holder desire to delegate to multiple DReps.
```

Comment:
Sorry, the conversion is a bit busy, which line exactly are you referring to? Nothing is preventing SPOs from being dReps, but nothing forces it either.

Reply:
I am referring to the line

> This implies that the DReps can (and likely will) differ from SPOs, so creating balance.

which in turn is referring to the previous line

> Rather than defining a third set of credentials, we instead propose to re-use the existing delegation credentials, using a new on-chain certificate to determine the governance stake distribution.

But I just can't see what is implied here. As I stated above, if anything, this implies that SPOs, who are already registered within the Cardano eco-system - and thus familiar with it, that probably either has a social following, or capital, and maybe even are connected and well informed, will be more likely to also be dReps.

Reply:
there might be a non-trivial intersection, that's fine, but I imagine a lot of DReps will not be SPOs. I can remove the work "likely" since really the numbers no one can guess for sure.